src/generic/datavgen.cpp

```diff
@@ -3275,6 +3302,8 @@ void wxDataViewMainWindow::SendSelectionChangedEvent( const wxDataViewItem& item
 
 void wxDataViewMainWindow::RefreshRow( unsigned int row )
 {
+    if (row == ((unsigned)-1))
```

Comment:
Do we really want to allow calling `RefreshRow(-1)`? Where does it happen?

I.e. the question is whether we shouldn't assert that the row is valid (`< total`) instead of silently returning here?